Examples/moltin tvOS Example/CategoriesViewController.swift

```diff
@@ -58,8 +58,8 @@ class ProductCategory: moltin.Category {
     var backgroundImage: String?
 
     enum ProductCategoryCodingKeys: String, CodingKey {
-        case backgroundColor = "background_colour"
-        case backgroundImage = "background_image"
+        case backgroundColor = "backgroundColor"
```

Comment:
This is in the flow as "background_colour" (english spelling) so the string should be:
```suggestion
        case backgroundColor = "backgroundColour"
```